### PR TITLE
Enable random_channel_buffer_tests to run on Linux

### PR DESCRIPTION
**Pull request by kain88-de**
With this patch applied I can see the following line in the test output:

   test tests::random_channel_buffer_tests::test_random_channel_buffers has been running for over 60 seconds

Without it I run in a failure with 'Invalid keystroke `cmk-,`

Release Notes:

- N/A

### CODE REVIEW

File: crates/collab/src/tests/test_server.rs [hunk 1]
```diff
@@ -271,6 +271,12 @@ impl TestServer {
             node_runtime: FakeNodeRuntime::new(),
         });
 
+        let os_keymap = if cfg!(target_os = "linux") {
+            "keymaps/default-linux.json"
+        } else {
+            "keymaps/default-macos.json" // Default case if none of the above
```
Comment (maxdeviant) [inline crates/collab/src/tests/test_server.rs:277]: I don't think we need this comment here:

```suggestion
            "keymaps/default-macos.json"
```

**Comment by kain88-de** [comment]
When do you plan to run the unit tests on linux?

**Comment by maxdeviant** [comment]
> When do you plan to run the unit tests on linux?

Probably as we get closer to a full Linux release.

I'm not sure what the current state of the tests is.

**Comment by kain88-de** [comment]
Currently 38 tests fail with this patch applied. I assume most of those are similar issues. I can work a bit on fixing them. I would feel better about that if the CI runs the tests to ensure they are not broken again in the future. Ideally all new test would also pass on Linux. 

However, do you want this at the moment?

**Comment by maxdeviant** [comment]
> Currently 38 tests fail with this patch applied. I assume most of those are similar issues. I can work a bit on fixing them. I would feel better about that if the CI runs the tests to ensure they are not broken again in the future. Ideally all new test would also pass on Linux.
> 
> However, do you want this at the moment?

I'm not sure that we're ready to enable Linux tests in CI quite yet.

**Pull request merged**